yarn.lock

```diff
@@ -645,85 +645,120 @@
     "@eslint-community/regexpp" "^4.10.0"
-    "@typescript-eslint/scope-manager" "8.0.0-alpha.34"
-    "@typescript-eslint/type-utils" "8.0.0-alpha.34"
-    "@typescript-eslint/utils" "8.0.0-alpha.34"
-    "@typescript-eslint/visitor-keys" "8.0.0-alpha.34"
+    "@typescript-eslint/scope-manager" "8.0.0-alpha.39"
+    "@typescript-eslint/type-utils" "8.0.0-alpha.39"
+    "@typescript-eslint/utils" "8.0.0-alpha.39"
+    "@typescript-eslint/visitor-keys" "8.0.0-alpha.39"
     graphemer "^1.4.0"
     ignore "^5.3.1"
     natural-compare "^1.4.0"
     ts-api-utils "^1.3.0"
 
-"@typescript-eslint/parser@8.0.0-alpha.34":
-  version "8.0.0-alpha.34"
-  resolved "https://registry.yarnpkg.com/@typescript-eslint/parser/-/parser-8.0.0-alpha.34.tgz#9a870a9986135a318ec9133f1fca19d9cd520ac7"
-  integrity sha512-jtBWP09o/RrVsLhDwoxUHtvJURZ7RaO3Ia9OnkC6Jjsdn23tKwoEtjLbB94ATr6BU44R3JVbRJn1SCueCmECYw==
+"@typescript-eslint/parser@8.0.0-alpha.39":
+  version "8.0.0-alpha.39"
+  resolved "https://registry.yarnpkg.com/@typescript-eslint/parser/-/parser-8.0.0-alpha.39.tgz#33fbd1e3767b4477def582ce597b6cd09bb5ef11"
+  integrity sha512-5k+pwV91plJojHgZkWlq4/TQdOrnEaeSvt48V0m8iEwdMJqX/63BXYxy8BUOSghWcjp05s73vy9HJjovAKmHkQ==
   dependencies:
-    "@typescript-eslint/scope-manager" "8.0.0-alpha.34"
-    "@typescript-eslint/types" "8.0.0-alpha.34"
-    "@typescript-eslint/typescript-estree" "8.0.0-alpha.34"
-    "@typescript-eslint/visitor-keys" "8.0.0-alpha.34"
+    "@typescript-eslint/scope-manager" "8.0.0-alpha.39"
+    "@typescript-eslint/types" "8.0.0-alpha.39"
+    "@typescript-eslint/typescript-estree" "8.0.0-alpha.39"
+    "@typescript-eslint/visitor-keys" "8.0.0-alpha.39"
     debug "^4.3.4"
 
-"@typescript-eslint/scope-manager@8.0.0-alpha.34":
-  version "8.0.0-alpha.34"
-  resolved "https://registry.yarnpkg.com/@typescript-eslint/scope-manager/-/scope-manager-8.0.0-alpha.34.tgz#8b08fdbc9d1d3bfe723d73d5eb0e7b67418c7d5b"
-  integrity sha512-IpeT8JnV1Uo5lG/GTYe/SRJRcz1rBaCNma5cS5R8c4NkBIiIeE+R9Vy8ZEPkGImTfBp9BUNU6w+8lSQf0Z6tKw==
+"@typescript-eslint/scope-manager@8.0.0-alpha.39":
+  version "8.0.0-alpha.39"
+  resolved "https://registry.yarnpkg.com/@typescript-eslint/scope-manager/-/scope-manager-8.0.0-alpha.39.tgz#1778198dcf95175b76631f7ffd84917bf925a1a9"
+  integrity sha512-HCBlKQROY+JIgWolucdFMj1W3VUnnIQTdxAhxJTAj3ix2nASmvKIFgrdo5KQMrXxQj6tC4l3zva10L+s0dUIIw==
   dependencies:
-    "@typescript-eslint/types" "8.0.0-alpha.34"
-    "@typescript-eslint/visitor-keys" "8.0.0-alpha.34"
+    "@typescript-eslint/types" "8.0.0-alpha.39"
+    "@typescript-eslint/visitor-keys" "8.0.0-alpha.39"
 
-"@typescript-eslint/type-utils@8.0.0-alpha.34":
-  version "8.0.0-alpha.34"
-  resolved "https://registry.yarnpkg.com/@typescript-eslint/type-utils/-/type-utils-8.0.0-alpha.34.tgz#c368ccd8c06d3cc4125ba94572d14bdd7eebaf74"
-  integrity sha512-VmsfGVQ9UV1gs+LQkA9W9Nf7rSwY9KzB7WZUXwx56Ynlwjyt+999Z4Rrh2kPuDCPHTsO+GJDqeYyOYOEeXi9Bw==
+"@typescript-eslint/scope-manager@8.0.0-alpha.40":
```

Comment:
their publishing script failed
https://github.com/typescript-eslint/typescript-eslint/actions/runs/9745098692/job/26893208631

Reply:
alpha.40 was only partially released, so now it's all out of sync

Reply:
looks like it's not going to fixed ASAP, so fixed the version to 39 in 7884eac